cmd/stop.go

```diff
@@ -30,13 +33,39 @@ var StopCmd = &cobra.Command{
 	Example: `
 # Stop Dapr application
 dapr stop --app-id <ID>
+
+# Stop multiple apps by providing a run config file
+dapr stop --run-file dapr.yaml
+
+# Stop multiple apps by providing a directory path containing the run config file(dapr.yaml)
+dapr stop --run-file /path/to/directory
 `,
 	Run: func(cmd *cobra.Command, args []string) {
+		var err error
+		if len(runFilePath) > 0 {
+			if runtime.GOOS == string(WINDOWS) {
+				print.FailureStatusEvent(os.Stderr, "The stop command with run file is not supported on Windows")
```

Comment:
```suggestion
				print.FailureStatusEvent(os.Stderr, "Stop command with run file is not supported on Windows")
```

Reply:
Can we also add a tracking issue here if it's in the eventual roadmap?

Reply:
It will be tracked as per the feature impl for windows